documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md

```diff
@@ -0,0 +1,138 @@
+#  Agenda and Minutes of CAMARA Technical Steering Committee (TSC) meeting - 2023-10-19
+
+## Invited TSC Participants
+
+* [x] Herbert Damker - (Deutsche Telekom AG) Active Maintainer, TSC chair
+* [x] Eric Murray - (Vodafone) Active Maintainer, TSC deputy chair
+* [x] Ludovic Robert - (Orange) Active Maintainer, TSC deputy chair
+* [x] Adnan Saleem - (Radisys) End User Council representative
+* [ ] Chris Howell - (Vonage) Active Maintainer
+* [x] Diego González Martínez - (Telefónica) Active Maintainer
+* [x] Doug Makishima - (Summit Tech) End User Council representative
+* [ ] George Glass - TM Forum representative
+* [ ] Henry Calvert - GSMA representative
+* [x] Mark Cornall - GSMA representative
+* [x] Jan Friman - (Ericsson) Active Maintainer
+* [x] Jose Luis Urien - (Telefonica) Active Maintainer
+* [ ] Kevin Smith - (Vodafone) Active Maintainer
+* [x] Mahesh Chapalamadugu - (Verizon) End User Council representative
+* [x] Shilpa Padgaonkar - (Deutsche Telekom AG) Active Maintainer
+* [x] Toshi (Toshiyasu) Wakayama - (KDDI) Active Maintainer
+
+Guest Presenter:
+
+* David McBride (Linux Fundation)
+
+Attendees:
+*  Sylvain Morel (Orange)
+*  Helene Vigue (GSMA)
+*  Ivan Izaquirre MArtin (Telefonica)
+*  Dinesh Muthukumar
+*  Mehdi Jafarizadeh
+*  Pierre Close (AT&T)
+*  Casey Cain (Linux Fundation)
+*  Tanja de Groot (Nokia)
+
+
+Minutes are available within: [https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC](https://github.com/camaraproject/Governance/tree/main/documentation/MeetingMinutes/TSC)
+
+## Agenda
+
+* Introduction
+* Release Management (#82) 
+* Commonalities & ICM 
+    * Alignment around securitySchemes and security of CAMARA API specs
+* API Backlog 
+    * New proposal for review "Network Insights"
+    * Decision on API Family for "Device Swap"
+    * Status of Site-to-cloud VPN proposal
+* (Further) open issues in https://github.com/camaraproject/Governance/issues
+* AOB
+
+Please find material for the agenda points below.
+
+## Introduction
+
+The participation of TSC Members covers well all CAMARA subproject.
+
+## Release Management
+
+A deck was prepared by Ludovic/Herbert (Available within meeting notes folder)
+
+Following points were discussed:
+
+**Discussion on API versioning** – DT proposal to use alpha<n> in API path until and API (major) version is stable
+  * Distinction between repostiory release (semantic version as before) and the version shown within API base path
+    * A sub project repository can contain one or multiple (closely related) APIs
+    * API has not to have the same version that the family release , and one of the proposal was that the release could match the most advanced API in sub-project. E.g.: Release 3.0 with API-X in version 3.0, API-Y in version v2alpha1 plus in version v1.0.2, etc)"

```

Comment:
"API has not to have" is too strong here - it is the equivalent of the 3GPP "API shall not have", when an API could have the same version number as the family release if things work out that way.

I'd suggest to reword:
"Each API has its own version number which may be different to that of the family release. One proposal was that ..."

Reply:
Agreed with @eric-murray  proposal
@shilpa-padgaonkar as you provided me the sentence ok for you?
Thanks

Reply:
@bigludo7 : This part of the sentence was not added/changed by me, I just changed the later half of it :)
But I am fine with the change proposed by Eric.